src/index.tsx

```diff
@@ -253,9 +255,11 @@ render(
         <DndProvider backend={HTML5Backend}>
           <Provider store={store}>
             <ReactReduxFirebaseProvider {...rrfProps}>
-              <AuthIsLoaded>
-                <AppRouter />
-              </AuthIsLoaded>
+              <CustomLoadersProvider>
+                <AuthIsLoaded>
+                  <AppRouter />
+                </AuthIsLoaded>
+              </CustomLoadersProvider>
```

Comment:
[note] This connects the our `CustomLoadersProvider` react context so that the `useCustomLoaders` hook can read from it.

Reply:
(backlogged to document properly in https://github.com/sparkletown/internal-sparkle-issues/issues/568)